MMM-voice.js

```diff
@@ -220,10 +222,51 @@ Module.register('MMM-voice', {
             MM.getModules().enumerate((module) => {
                 module.hide(1000);
             });
+            // tell other modules all hidden
+            this.sendNotification('NOW_ASLEEP','[]')
         } else if (notification === 'SHOW') {
             MM.getModules().enumerate((module) => {
                 module.show(1000);
             });
+            // tell other modules all shown
+            this.sendNotification('NOW_AWAKE')
```

Comment:
The following notifications are all in the same topic (stand by status). What you basically do is creating multiple notifications which have the information in the name, but it should be one notification, that has the information in the payload. 

```
STAND_BY_STATUS: {
  type: 'hide', // hide, show, ...
  hardware: true, //true, false
}
```

Reply:
and I thought about using one notice with different data, but felt that was too restrictive

Reply:
I dont think its more restrictive, it even gives you more options, as you can put as many details as you want into the payload instead of adding another and another notification. Also it helps to integrate something more generic to other modules. If a notification is only restricted to a specific module and then you want to support multiple modules, the code gets a mess because you need to integrate 20 notifications instead of 2-3...